### PR TITLE
Sdl-gl begin making some functions

### PR DESCRIPTION
**Pull request by adam000**
Just the bare minimum needed to draw polys to the screen

### CODE REVIEW

File: sdl/gl.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package sdl
+
+// #cgo pkg-config: sdl2
```
Comment (adam000) [inline sdl/gl.go]: Do I need this? I don't think so...

File: sdl/gl.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+//
+// #include "SDL.h"
+import "C"
+
+type GLContext struct {
+	c *C.SDL_GLContext
```
Comment (zombiezen) [inline sdl/gl.go]: This shouldn't be a pointer.  SDL_GLContext is a `void*` pointer underneath.

File: sdl/gl.go (hunk 1 above)
Comment (zombiezen) [inline sdl/gl.go]: No. I just went to great lengths to strip these redundant pkg-configs out.  Run `go install -x` to see just what this does.

**Comment by zombiezen** [comment]
Also, we should be trying to document all exported names as we go.

https://code.google.com/p/go-wiki/wiki/Style#Doc_Comments

**Comment by adam000** [comment]
I did the thing

**Pull request merged**